svgs/milanote.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="192" height="192" viewBox="0 0 192 192"><defs><style>.cls-1{fill:none;}</style></defs><path d="M176,170.36h0a5.65,5.65,0,0,1-5.64,5.64h-1a5.63,5.63,0,0,1-4-1.65L17.65,26.63a5.63,5.63,0,0,1-1.65-4v-1a5.64,5.64,0,0,1,9.63-4L96,88l8.48,8.48L164,156l8,8,2.37,2.37A5.63,5.63,0,0,1,176,170.36Z"/><path d="M176,21.64v1a5.63,5.63,0,0,1-1.65,4L26.63,174.35a5.63,5.63,0,0,1-4,1.65h-1A5.65,5.65,0,0,1,16,170.36h0a5.63,5.63,0,0,1,1.65-4L20,164l8-8L87.52,96.5,96,88l70.37-70.37A5.64,5.64,0,0,1,176,21.64Z"/><path d="M28,22.36v148A5.65,5.65,0,0,1,22.36,176h-.72A5.65,5.65,0,0,1,16,170.36V21.64a5.64,5.64,0,0,1,9.63-4l.72.72A5.63,5.63,0,0,1,28,22.36Z"/><path d="M176,21.64V170.36a5.65,5.65,0,0,1-5.64,5.64h-.72a5.65,5.65,0,0,1-5.64-5.64v-148a5.63,5.63,0,0,1,1.65-4l.72-.72A5.64,5.64,0,0,1,176,21.64Z"/><rect x="16" y="164" width="160" height="12" rx="5.64"/><rect class="cls-1" width="192" height="192"/></svg>
```

Comment:
I think you could greatly simplify your svg by using `LineTo` like so:

```svg
<svg xmlns="http://www.w3.org/2000/svg"
    fill="none"
    stroke="#00e1ff"
    stroke-width="12"
    stroke-linejoin="round"
    width="192"
    height="192"
    viewBox="0 0 192 192">

    <path d="M 21.92,170.233 170.105,170.111 21.764062,22.181621 21.92,170.233 170.29722,22.092618 170.105,170.111" />
</svg>
```

(Note that the coordinates may not be perfect, I have just overlapped your icon without really looking at the numbers)

Reply:
Alright. I changed it to the simplified code. Thanks!

What do you use to make the icons?? I use Illustrator but kind find how to convert them into lines

Reply:
I just started using Inkscape because that's what ``@ThatOneCalculator`` mentioned in #546. But I am not a designer, so I still find myself opening the SVGs in vscode to get a more "nerdy" view 😅

Reply:
Ahh alright. I tried using Inkscape but looks so ancient to me 😆

Reply:
> Ahh alright. I tried using Inkscape but looks so ancient to me :laughing:

How so? Inkspace, especially with LibAdwaita, looks pretty modern. 
![image](https://user-images.githubusercontent.com/44733677/173881696-33c985df-ad19-4307-8046-e2bd8991baec.png)